java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S107_java.html

```diff
@@ -14,6 +14,11 @@ <h2>Compliant Solution</h2>
 }
 </pre>
 <h2>Exceptions</h2>
-<p>Methods annotated with Spring's <code>@RequestMapping</code> (and related shortcut annotations, like <code>@GetRequest</code>) or
-<code>@JsonCreator</code> may have a lot of parameters, encapsulation being possible. Such methods are therefore ignored.</p>
+<p>Methods annotated with :</p>
+<p> * Spring's <code>@RequestMapping</code> (and related shortcut annotations, like <code>@GetRequest</code>)</p>
+<p> * JAX-RS API annotations (like <code>@javax.ws.rs.GET</code>)</p>
+<p> * Bean constructor injection with <code>@org.springframework.beans.factory.annotation.Autowired</code></p>
+<p> * CDI constructor injection with <code>@javax.inject.Inject</code></p>
+<p> * <code>@com.fasterxml.jackson.annotation.JsonCreator</code></p>
```

Comment:
This should be an HTML list.
It's a ruleapi bug, you have to remove the space before the `*` in the description.

I already did it, you just have to generate the metadata to solve the problem.

Reply:
Thanks, didn't know about it 👍